introspection-engine/connectors/sql-introspection-connector/src/sanitize_datamodel_names.rs

```diff
@@ -82,7 +82,7 @@ fn sanitize_models(datamodel: &mut Datamodel, family: &SqlFamily) -> HashMap<Str
                                 let sanitized_value = sanitize_string(value);
 
                                 match sanitized_value {
-                                    x if x == EMPTY_STRING => DefaultValue::new_db_generated(),
+                                    x if x == EMPTY_STRING => DefaultValue::new_db_generated("".to_string()),
```

Comment:
now that we actually migrate these, that is a problem since we'd now change the default value

Reply:
we could just put the unsanitized value in there.